docs/content/Configuration/VPC/Connecting-with-a-VPC-AWS.mdx

```diff
@@ -15,57 +15,96 @@ required:
 - **AWS VPC ID:** The ID of the VPC that Cube Cloud will connect to, for example
   `vpc-0099aazz`
-- **AWS VPC CIDR:** The CIDR block of the VPC that Cube Cloud will connect to,
-  for example `10.0.0.0/16`
+- **AWS VPC CIDR:** The [CIDR block][wiki-cidr-block] of the VPC that Cube Cloud
+  will connect to, for example `10.0.0.0/16`
+
+After receiving the information above, a Customer Success Manager will provide
+you with an **AWS account ID**, **region**, a **VPC ID** and a [**CIDR
+block**][wiki-cidr-block] used by Cube Cloud to connect to your VPC.
 
 ## Setup
 
-### VPC Peering Request
+### <--{"id" : "Setup"}--> VPC Peering Request
 
 After receiving the information above, Cube Cloud will send a [VPC peering
 request][aws-docs-vpc-peering] that must be accepted. This can be done either
 through the [AWS Web Console][aws-console] or through an infrastructure-as-code
 tool.
 
-To accept the VPC peering request through the AWS Web Console, follow [the
-instructions here][aws-docs-vpc-peering-accept] with the following amendments:
+To [accept the VPC peering request][aws-docs-vpc-peering-accept] through the AWS
+Web Console, follow the instructions below:
+
+<WarningBox>
+
+Ensure the peering request is from Cube Cloud by checking that the **AWS account
+ID**, **region** and **VPC IDs** match those provided by Support.
+
+</WarningBox>
+
+1. Open the Amazon VPC console at https://console.aws.amazon.com/vpc/.
 
-- On Step 4, verify the peering request is from Cube Cloud by checking that the
-  AWS account ID, region and VPC IDs match those provided by Support.
-- On Step 5, ensure **Modify my route tables now** is selected so that the
-  necessary routes are created.
-- After Step 5, the security group for any databases within the VPC
-  [may require updating](#updating-security-groups) to allow traffic from Cube
-  Cloud.
+2. In the navigation pane, choose <Btn>Peering connections</Btn>.
 
-### Updating security groups
+3. Choose <Btn>Create peering connection</Btn>.
 
-Often the initial VPC setup will not allow traffic from Cube Cloud; this is
-usually because [the security group][aws-docs-vpc-security-group] for the
-database will need to allow access from the Cube Cloud CIDR block.
+4. Configure the information as follows, and choose <Btn>Create peering
+   connection</Btn> when you are done:
+   - **Name**: You can optionally name your VPC peering connection. Doing so
+     creates a tag with a key of `Name` and a value that you specify. This tag
+     is only visible to _you_.
+   - **VPC ID (Requester)**: Select the VPC in your account with which to create
+     the VPC peering connection.
+   - **Account**: Choose <Btn>Another account</Btn>.
+   - **Account ID**: Enter the ID of the Cube Cloud AWS account as provided by
+     Support.
+   - **Region**: Choose <Btn>Another region</Btn> and select the Region as
+     provided by Support.
+   - **VPC ID (Accepter)**: Enter the ID of the Cube Cloud VPC as provided by
+     Support.
```

Comment:
I'm sorry, is this correct? I'm not an AWS or VPC expert, but it looks like the steps to "create a VPC peering request" rather than to accept one. Also, by this link (https://docs.aws.amazon.com/vpc/latest/peering/accept-vpc-peering-connection.html), I see different steps.